projects/Flarum/project.nix

```
@@ -0,0 +1,7 @@
+{pkgs, ...}: {
+  packages =
+    if pkgs == {}
+    then builtins.throw "wut?"
+    else {inherit (pkgs) flarum;};
```

```suggestion
  packages = {inherit (pkgs) flarum;};
```

Probably no need to throw?

Also that could as well be `default.nix` because it's the entry point to this piece of code.

---

Yes, that's me debugging. Sorry.